deployment/v3/mosip/mosipcertmanager/install.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+# Installs sample print service
+## Usage: ./restart.sh [kubeconfig]
+
+if [ $# -ge 1 ] ; then
+  export KUBECONFIG=$1
+fi
+
+
+NS=mosipcertmanager
+CHART_VERSION=12.0.2
```

Comment:
please change the version to 0.0.1-develop

Reply:
Updated Chart version hence closing this Conversation